packages/main/src/RPA/JSON.py

```diff
@@ -9,13 +9,98 @@
 
 
 class JSON:
-    """`JSON` is a library for manipulating `JSON`_ objects.
-    Locating specific elements in the structure is done using `JSONPath`_.
+    """`JSON` is a library for manipulating `JSON`_ files and strings.
+
+    **About JSONPath**
+
+    Reading and writing values to JSON objects is done using `JSONPath`_.
+    It's a syntax designed to quickly and easily refer to specific
+    elements in a JSON structure.
+
+    Compared to Python's normal dictionary access, JSONPath expressions can
+    target multiple elements through features such as conditionals and wildcards,
+    which can simplify many JSON-related operations. It's analogous to XPath
+    for XML structures.
 
     .. _JSON: http://json.org/
     .. _JSONPath: http://goessner.net/articles/JsonPath/
+
+    **Syntax example**
+
+    For this example consider the following structure:
+
+    .. code-block:: json
+
+       {
+         "clients": [
+           {
+             "name": "Johnny Example",
+             "email": "john@example.com",
+             "orders": [
+                 {"address": "Streetroad 123", "price": 103.20},
+                 {"address": "Streetroad 123", "price": 98.99}
+             ]
+           },
+           {
+             "name": "Jane Example",
+             "email": "jane@example.com",
+             "orders": [
+                 {"address": "Waypath 321", "price": 22.00},
+                 {"address": "Streetroad 123", "price": 2330.01}
+             ]
+           }
+         ]
+       }
+
+    In the simplest case JSONPath can replace nested access:
+
+    .. code-block:: robotframework
+
+       # First order of first client, with direct dictionary access
+       ${value}=    Set variable    ${json}["clients"][0]["orders"][0]
+
+       # JSONPath access
+       ${value}=    Get value from JSON    ${json}    $.clients[0].orders[0]
+
```

Comment:
This got me thinking... I like Python doctest style usage examples
```>>> sum([1,2,3])
6
```
Does Robot Framework documentation have a tradition in these kinds of examples?

Reply:
Not exactly like that. I've tried to display it in keyword form (at least in some other examples). There's a wish/requirement that the examples can be copy-pasted easily, and quite often keywords are more about side-effects than return values. Maybe it could be as a comment below it, if relevant?